blog/2024-03-04-config-entry-reconfigure-step.md

```diff
@@ -0,0 +1,44 @@
+---
+author: G Johansson
+authorURL: https://github.com/gjohansson-ST
+title: "Config Entries can now provide a reconfigure step"
+---
+
+As of Home Assistant Core 2024.4 config entries can now be reconfigured by adding a `reconfigure` step in their config flows.
+
+This is not to replace optional configuration (`OptionsFlow`) but instead to allow the user to change setup configuration after a config entry has been created.
+
+### Integrations updating existing config entries
+
+Some integrations implement a reconfiguration option by updating an existing config entry if a matching config entry is being created by the user. This is no longer allowed, and integrations should be migrated to implement the `reconfigure` step.
```

Comment:
We should really publish this blog post, and the discussion about the wording here is blocking. Let's remove this for now since very few integrations implement the implicit reconfigure.

```suggestion
```